doc/install/index.rst

```diff
@@ -40,16 +39,20 @@ Other methods
 Ceph clusters using Ansible.
 
 * ceph-ansible is widely deployed.
-* ceph-ansible is not integrated with the new orchestrator APIs,
-  introduced in Nautilus and Octopus, which means that newer
-  management features and dashboard integration are not available.
+* ceph-ansible is not integrated with the orchestrator APIs introduced in
+  Nautilus and Octopus, which means that the management features and dashboard
+  integration they introduced are not available in Ceph clusters deployed by
```

Comment:
which means that these API management features and dashboard integration are not available in Ceph clusters

To me the antecedent of "they" is ambiguous here.

Reply:
Fair enough. I'll recast. it.